main.js

```diff
@@ -0,0 +1,21 @@
+const Controller = () => {
+    const view = View()
+    const model = Model()
+
+    const loaduser = async () => {
```

Comment:
it is an async function, but you do not need to put async here. you will learn how to use async await later